data/core/config.lua

```diff
@@ -3,7 +3,7 @@ local config = {}
 config.project_scan_rate = 5
 config.fps = 60
 config.max_log_items = 80
-config.message_timeout = 3
+config.message_timeout = 5
```

Comment:
I think this is a more sensible default but as it is configurable I can change it back if you want.

Reply:
I agree, the previous value was a bit too fast.